addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1028,6 +1028,90 @@ registry.HeaderHamburgerFull = publicWidget.Widget.extend({
     },
 });
 
+registry.BottomFixedElement = publicWidget.Widget.extend({
+    selector: '#wrapwrap',
+    disabledInEditableMode: false,
+
+    /**
+     * @override
+     */
+    start() {
+        this.$target.on('scroll.bottom_fixed_element', _.debounce(this._hideBottomFixedElements.bind(this), 100));
+        $(window).on('resize.bottom_fixed_element', _.debounce(this._hideBottomFixedElements.bind(this), 100));
+        return this._super(...arguments);
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        this._super(...arguments);
+        this.$target.off('.bottom_fixed_element');
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * Hide the element that is fixed at the bottom of the screen if the scroll
+     * reaches the bottom of the page and if this element hides a button.
+     *
+     * @private
+     */
+    _hideBottomFixedElements() {
```

Comment:
Handlers should begin by `_on` and placed after the "Private" section.

Reply:
it's more a private function as `_computeHeights()` or `_adaptSize()` (in the same file)
so a didnt add `_on` but move it in the private section